internal/indexer/definitions/torrentseeds.yaml

```diff
@@ -0,0 +1,65 @@
+---
+#id: TorrentSeeds
+name: TorrentSeeds
+identifier: torrentseeds
+description: TorrentSeeds (TS) is a GENERAL/0-DAY tracker with great pretimes.
+language: en-us
+urls:
+  - https://torrentseeds.org
+privacy: private
+protocol: torrent
+supports:
+  - irc
+  - rss
+source: custom
+settings:
+  - name: passkey
+    type: secret
+    label: Passkey
+    help: "Go to General / Get RSS / Check a category and click create your passkey is after torrent_pass="
+
+irc:
+  network: Torrentseeds.org
+  server: irc.torrentseeds.org
+  port: 6697
+  tls: true
+  channels:
+    - "#announce"
+  announcers:
+    - torrentseeds
+  settings:
+    - name: nickserv.account
+      type: text
+      required: false
+      label: NickServ Account
+      help: NickServ account. Make sure to group your user and bot. Eg. user_bot
+    - name: nickserv.password
+      type: secret
+      required: false
+      label: NickServ Password
+      help: NickServ password
+    - name: invite_command
+      type: secret
+      default: "Cerberus identify username irckey"
+      required: true
+      label: Invite command
+      help: Invite auth with Cerberus.

```

Comment:
Add after the dot `Replace USERNAME and IRCKEY`.

Reply:
added on my side